src/network/netdev/tunnel.c

```diff
@@ -46,19 +46,29 @@ static const char* const ip6tnl_mode_table[_NETDEV_IP6_TNL_MODE_MAX] = {
 DEFINE_STRING_TABLE_LOOKUP(ip6tnl_mode, Ip6TnlMode);
 DEFINE_CONFIG_PARSE_ENUM(config_parse_ip6tnl_mode, ip6tnl_mode, Ip6TnlMode, "Failed to parse ip6 tunnel Mode");
 
+static const char* const create_type_table[_NETDEV_CREATE_MAX] = {
+        [NETDEV_CREATE_INDEPENDENT] = "indepedent",
+        [NETDEV_CREATE_STACKED] = "stacked",
```

Comment:
why make this an enum if it only has two values? Do you expect more values to be added later?

It feels to me this should just be a bool called "Independent=yes|no" or so?

Reply:
Yes . updated.